src/corner_kick/jest.config.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  "roots": [
+    "<rootDir>/src"
+  ],
+  "transform": {
+    "^.+\\.tsx?$": "ts-jest"
```

Comment:
Maybe some documentation here would be useful. What does this line of code mean?

Reply:
Added an additional comment.